spec.md

```diff
@@ -1796,7 +1796,11 @@ The short syntax uses a single string with colon-separated values to specify a v
 (`VOLUME:CONTAINER_PATH`), or an access mode (`VOLUME:CONTAINER:ACCESS_MODE`).
 
 `VOLUME` MAY be either a host path on the platform hosting containers (bind mount) or a volume name.
-`ACCESS_MODE` MAY be set as read-only by using `ro` or read and write by using `rw` (default).
+`ACCESS_MODE` is a comma-separated `,` list of options and MAY be set as read-only by using `ro`,
```

Comment:
Could you please turn it into bullet points (or otherwise break it up) for better readability?